practice-app/backend/api/urls.py

```diff
@@ -1,10 +1,19 @@
 from django.urls import path,include
-from api.views import country_views
+from api.views import country_views,sport_views
 from api.routers.country_router import country_router
+from api.routers.sport_router import league_season_router,standing_router
+from rest_framework import routers
+
+
+router = routers.DefaultRouter()
+router.registry.extend(country_router.registry)
+router.registry.extend(league_season_router.registry)
+router.registry.extend(standing_router.registry)
 
 urlpatterns = [
-    path('', include(country_router.urls)),
-    path("countries/new/", country_views.CreateCountry.as_view()),
-    path("countries/clearall/", country_views.ClearCountries.as_view()),
-    path("countries/collect/", country_views.CollectData.as_view()),
+    path('', include(router.urls)),
+    path("countries_new/", country_views.CreateCountry.as_view()),
+    path("countries_clearall/", country_views.ClearCountries.as_view()),
+    path("countries_collect/", country_views.CollectData.as_view()),
+    path("league_seasons_collect/",sport_views.CollectData.as_view()),
```

Comment:
böyle yapmaya devam edin eğer hala hata veriyorsa ama zamanım kalırsa bakmaya çalışacağım böyle yapmamak lazım.

Reply:
Yani şu anda sanırsam araya '/' koyunca farklı url dosyasına geçmeyi bekliyor onun için doğru view a ulaşamıyor. Bunun için issue açılabilir ve bir çözüm yolu araştırılabilir. Ama şimdilik böyle kalsın.